### PR TITLE
Prevents posts from reloading when changing the comments sorting

### PR DESCRIPTION
**Pull request by rodrigo-fm**
## Description

When we sort the comments on the PostActivity, the post information is also fetched (instead of just the comments). This PR fixes this. 

### Before

https://github.com/dessalines/jerboa/assets/56604262/d09796f6-a7df-4522-9f95-5d9d64f31275

### After

https://github.com/dessalines/jerboa/assets/56604262/6670a095-4a15-4523-aa9a-459758466d37

### CODE REVIEW

**Review by MV-GH — CHANGES_REQUESTED** [review]
Thanks! run `./gradlew formatKotlin` to pass the linter

File: app/src/main/java/com/jerboa/ui/components/post/PostActivity.kt [hunk 1]
```diff
@@ -207,33 +207,33 @@ fun PostActivity(
...
-                    if (navigateParentCommentsWithVolumeButtons) {
-                        when (keyEvent.key) {
-                            Key.VolumeUp -> {
-                                scrollToPreviousParentComment(scope, parentListStateIndexes, listState)
-                                true
-                            }
+        Modifier
+            .nestedScroll(scrollBehavior.nestedScrollConnection)
+            .semantics { testTagsAsResourceId = true }
+            .nestedScroll(scrollBehavior.nestedScrollConnection)
```
Comment (MV-GH) [inline app/src/main/java/com/jerboa/ui/components/post/PostActivity.kt:213]: Just realized but this is double, can you remove the double line

Comment (rodrigo-fm) [reply app/src/main/java/com/jerboa/ui/components/post/PostActivity.kt]: removed

**Review by dessalines — APPROVED** [review]
Nice, thx!

**Review by MV-GH — APPROVED** [review]

**Pull request merged**